catapult-sdk/src/model/ModelType.js

```diff
@@ -42,8 +42,11 @@ const ModelType = {
 	/** Schema property type indicating a uint16. */
 	uint16: SchemaType.max + 5,
 
+	/** Schema property type indicating a uint32 value. */
+	uint32: SchemaType.max + 6,
```

Comment:
why was this added? any specific reason? it was a basic type, so it was not needed, i don't see any changes made to the formatting rules

Reply:
that's right, will remove